src/views/StandaloneDirectorsFiling.vue

```diff
@@ -738,76 +750,77 @@ export default {
     async fetchChangeOfDirectors (): Promise<void> {
       const url = `businesses/${this.entityIncNo}/filings/${this.filingId}`
       await axios.get(url).then(async response => {
-        if (response && response.data) {
-          const filing = response.data.filing
-          try {
```

Comment:
I got rid of this redundant try-catch becayse the next line verifies that `filing` exists (and throws if not).